src/lib/ng-packagr.ts

```diff
@@ -43,6 +43,7 @@ export async function createNgPackage(opts: NgPackagrCliArguments): Promise<void
     }
 
     await copyFiles(`${rootPackage.sourcePath}/README.md`, rootPackage.destinationPath);
+    await copyFiles(`${rootPackage.sourcePath}/CHANGELOG.md`, rootPackage.destinationPath);
```

Comment:
New "feat: copies `CHANGELOG.md` to package destination"

Reply:
Absolutely right, i committed this by mistake :)